tooling/operation/get-parameters-as-json-schema.js

```diff
@@ -197,6 +204,15 @@ function constructSchema(data, prevSchemas = [], currentLocation = '', globalDef
       schema.type = 'object';
     } else if ('items' in schema) {
       schema.type = 'array';
+    } else if (isPolymorphicAllOfChild) {
```

Comment:
Since this shapeless case only happens with an `allOf` where each schema within that block is merged together, we shouldn't apply this logic to `oneOf` and `anyOf` polymorphic schemas.

Reply:
am I reading this right that we're just leaving this with an empty if block?

Is this preferred over switching the final else to else if not polymporphic? I'm not sure I have an opinion just yet.

Reply:
Yeah this is just a no-op. Figured it was cleaner and easier to read this way.